datasetinsights/commands/train.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+import click
+
+import datasetinsights.constants as const
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    help="Start model training (and optionally validation) tasks.",
+    context_settings=const.CONTEXT_SETTINGS,
+)
+@click.option(
+    "-c",
+    "--config",
+    type=click.STRING,
```

Comment:
Can be changed to `click.File()`

Reply:
sams as in evaluate.py. Will not change this to click.File()